### ISSUE TITLE
Status column values do not change gantt chart task highlight color to indicate progress

### ISSUE THREAD

**Issue by aaraney** [panyam/ganttsheet#16]
Hi @panyam!

First off, thanks for writing and publishing this google sheets extension. Ive tested all the sheets extensions that come up when you query for "gantt" in the add-ons store, and yours is by far my favorite. Its simple and just works.

To preface this, if you are open to this feature and can point me in the right, i'm happy to open a PR and go through a review together to add this feature. Ive looked around the code base and have a general idea where the feature might be added, so i'm not flying blind.

The feature I propose is to use the `status` column's values to display a tasks progress by changing how the task is highlighted in the calendar view. For example, a status of 50% would mean the first half of the highlighted columns in that row in the calendar view would have a different color. Since the calendar view is highlighted at the day scale, meaning if a task lasted for two days then two columns in that row would be highlighted, I think it makes the most sense to round down for highlighting purposes. So for example, lets say the task has a duration of 1 day and the task is 50% complete, to calculate the number of columns to highlight in a progress color would be:

```js
let taskPeriodNDays = 1
let statusAsDecimal = 0.5

let nProgressBars = Math.floor(statusAsDecimal * taskPeriodNDays) // 0
let nRemainingBars = taskPeriodNDays - nProgressBars
```

I know there are limitations making this assumption, namely indicated task's progress length is a function of the task's length, however I still this it would be nice feature to add. Its worth noting that if a status column is not a percentage then this feature would be ignored and the current functionality would go unchanged. What are your thoughts?

**Comment by panyam** [comment]
Hey Austin thanks for reaching out mate.  I havent had a chance to keep this maintained though.   Are you able to test this on your own google sheets dev account? Definitely happy to use your help.

**Comment by aaraney** [comment]
Certainly! Yeah, no worries -- I figured that is one reason why you open-sourced the project. 

I am fairly new to the apps script ecosystem, but I figure I can get a development environment setup to start making some progress.
